lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -378,4 +422,4 @@ function dosomething_reportback_flag_confirm_submit($form, &$form_state) {
   }
   // Log the flag operation.
   $reportback->insertLog($op);
-}
+}
```

Comment:
Looks like you removed the blank line, https://www.drupal.org/node/318#indenting